webserver/src/main/java/io/kestra/webserver/controllers/api/FlowController.java

```diff
@@ -533,6 +538,53 @@ public List<ValidateConstraintViolation> validateFlows(
             .collect(Collectors.toList());
     }
 
+    @ExecuteOn(TaskExecutors.IO)
```

Comment:
```suggestion
    // This endpoint is not used by the Kestra UI nor our CLI but is provided for the API users for convenience
    @ExecuteOn(TaskExecutors.IO)
```

Adding a comment will prevent someone removing the method as not used ;)